Python/Product/PythonTools/PythonTools/Project/PythonInfoBar.cs

```diff
@@ -50,6 +48,10 @@ internal abstract class PythonInfoBar : IVsInfoBarUIEvents, IDisposable {
 
         public abstract Task CheckAsync();
 
+        protected virtual bool IsGloballySuppressed  =>
+            /// Needs to be called from mainthread
+            !Site.GetPythonToolsService().GeneralOptions.PromptForEnvCreate;
+
```

Comment:
this is the generic base class, it shouldn't be looking at any specific option.
it's also not called from anywhere in this class, so doesn't need to be defined here.
should be moved back to `CondaEnvCreateInfoBar`

Reply:
ok. will do

Reply:
done